### ISSUE TITLE
after inference, appear Weird female face

### ISSUE THREAD

**Issue by AWangji** [nihaomiao/WACV23_TSNet#19]
Hi, thanks for your great work. 
But when I replace your images in "00110" with my own images, and aiming to drive it dance. I have get the pose detection using openpose model and place the json files in the right directory. However, after I execute the demo_pose.py, a Weird female face appeared:
![image](https://github.com/nihaomiao/WACV23_TSNet/assets/42883258/97826c8e-674a-4fce-86c8-25ec85e69654)
why happened that? Could you please provide some suggestions?

**Comment by nihaomiao** [comment]
Hi, @AWangji, sorry for the late reply! The reason is overfitting. Due to the limitation of computational resources, our pretrained model was only trained on two subjects, which made it only suitable for the "seen" person applications (the man/woman I showed in the example videos). If you want to apply our model to your own images, you need some fine-tuning to make the model generalize to your photos. For example, use the video of yourself to finetune the model with self-reconstruction loss.
